src/components/SettingsSection/SettingsSection.vue

```diff
@@ -0,0 +1,75 @@
+<!--
+  - @copyright Copyright (c) 2019 Julius Härtl <jus@bitgrid.net>
```

Comment:
no? :)

Reply:
I adapted the SettingsSection from richdocuments that Julius did. So we are co-authors :) need to add mine too